dist/leaflet.distortableimage.js

```diff
@@ -699,13 +718,15 @@ L.DistortableImage.Edit = L.Handler.extend({
 			79: '_toggleOutline', // o
 			82: '_toggleRotateDistort', // r
       84: '_toggleTransparency', // t
+			88: '_toggleOrder', // x
```

Comment:
Here we could bind a `bringForward` and `...Back` or equivalent to the up arrow and down arrows?

Reply:
It seems that all the arrows are defaulted to map movement in the leaflet module. Should I override those wirings?